en/ui-system/components/editor/webview.md

```diff
@@ -0,0 +1,210 @@
+import { _decorator, Component, WebView } from 'cc';
+const { ccclass, property } = _decorator;
+
+@ccclass('WebViewCtrl')
+export class WebViewCtrl extends Component {
+    @property(WebView)
+    webview = null;
+
+    start () {
+        this.webview.node.on('loaded', this.callback, this);
+    }
+
+    callback (event) {
+       // The event here is an EventCustom object, and you can get the WebView component through event.detail
+        let videoplayer = event.detail;
+        // do whatever you want with webview
+        // Also, note that this way the registered event can not pass customEventData
+    }
+}
+```
+
+Likewise, you can also register `loading`, `error` events, and the parameters of the callback function for these events are consistent with the `loaded` parameters.
+
+## How to interact with WebView internal pages
+
+### Calling the WebView internal page
+
+```ts
+import { _decorator, Component, WebView } from 'cc';
+const { ccclass, property } = _decorator;
+
+@ccclass('WebViewCtrl')
+export class WebViewCtrl extends Component {
+    @property(WebView)
+    webview = null;
+
+    start () {
+        // The Test here is a global function defined in your webView's internal page code
+        this.webview.evaluateJS('Test()');
+    }
+}
+```
+
+#### Note: Cross domain issues on Web platform need to be resolved by yourself
+
+### WebView internal pages call external code
+
+At present, the mechanism of Android and IOS is to determine whether the key of URL prefix is the same as that of the URL prefix by intercepting the jump, and then callback if the same.
+
+1. Setting the URL prefix keyword through `setJavascriptInterfaceScheme`
+2. The callback function is set by `setOnJSCallback`, and the function parameter is URL
+
+```ts
+import { _decorator, Component, WebView } from 'cc';
+const { ccclass, property } = _decorator;
+
+@ccclass('WebViewCtrl')
+export class WebViewCtrl extends Component {
+    @property(WebView)
+    webview = null;
+
+    // Setting in onLoad will make the callback useless, so we must set the cc.WebView callback in the start cycle.
+    start () {
+        // Here are the keywords that are agreed with the internal page
+        // Please set the scheme with lower case, the native won't identify the uppercase char scheme.
+        let scheme = "testkey";
+
+        function jsCallback (target, url) {
+            // The return value here is the URL value of the internal page, and it needs to parse the data it needs.
+            let str = url.replace(scheme + '://', ''); // str === 'a=1&b=2'
+            // webview target
+            console.log(target);
+        }
+
+        this.webview.setJavascriptInterfaceScheme(scheme);
+        this.webview.setOnJSCallback(jsCallback);
+    }
+}
+```
+
+So when you need to interact with WebView through an internal page, you should set the internal page URL: `testkey://(the data you want to callback to WebView later)`. WebView internal page code:
```

Comment:
So when you need to interact with WebView through an internal page, you should set the internal page URL: `testkey://(the data you want to callback to WebView later)`. WebView internal page code:
-->
When you need to interact with __WebView__ through an internal page, you should set the internal page URL: `testkey://(the data you want to callback to WebView later)`. __WebView__ internal page code: